src/screens/Menu/SettingsSection.js

```diff
@@ -39,95 +38,57 @@ const SectionHeader = styled(MediumText)`
   margin-horizontal: ${spacing.layoutSides}px;
 `;
 
-const Separator = styled.View`
-  width: 100%;
-  height: 1px;
-  background-color: ${themedColors.border};
-`;
-
 type Props = {
-  sectionTitle: string,
+  sectionTitle?: string,
   sectionItems: Object[],
-  isCardsList?: boolean,
-}
+};
 
 const Section = (props: Props) => {
-  const { sectionItems, isCardsList } = props;
-  if (isCardsList) {
-    return (
-      <FlatList
-        keyExtractor={item => item.key}
-        data={sectionItems}
-        style={{ marginTop: -spacing.medium }}
-        contentContainerStyle={{ width: '100%', paddingHorizontal: spacing.layoutSides, paddingTop: spacing.medium }}
```

Comment:
Applying paddings here was necessary for shadowed cards. If paddings are applied on SettingsSection parent (as they are now), cards' shadows get clipped.